arches/app/datatypes/datatypes.py

```diff
@@ -1614,7 +1614,12 @@ def post_tile_save(self, tile, nodeid, request):
                 file_model.path = file_data
                 file_model.tile = tile
                 if models.TileModel.objects.filter(pk=tile.tileid).count() > 0:
+                    original_storage = file_model.path.storage
+                    # this temporary storage change _bypasses_ s3 storage temporarily
```

Comment:
Hey Aaron - just to clarify this a bit more, could you add another comment stating something like: "Prevents Django's file storage api from overwriting files uploaded directly from the client, re #9321"